### PR TITLE
learning space page is added

### PR DESCRIPTION
**Pull request by kamilkorkut**
I have added learning space page which shows current learning space information, and contains add content button and join button.

### CODE REVIEW

**Comment by hseyyidkaplan** [comment]
> 
Dear Kamil, It is encouraged and advised us to add a description and screenshots of newly developed parts in project with this pull request. For next times, it will be better to see them on behalf of you.

File: frontend/yarn.lock [hunk 1]
```diff
@@ -175,7 +175,7 @@
     "core-js-pure" "^3.25.1"
     "regenerator-runtime" "^0.13.10"
 
-"@babel/runtime@^7.10.2", "@babel/runtime@^7.12.5", "@babel/runtime@^7.18.3", "@babel/runtime@^7.18.9", "@babel/runtime@^7.20.1", "@babel/runtime@^7.5.5", "@babel/runtime@^7.8.7":
+"@babel/runtime@^7.10.2", "@babel/runtime@^7.12.5", "@babel/runtime@^7.17.2", "@babel/runtime@^7.18.3", "@babel/runtime@^7.18.9", "@babel/runtime@^7.20.1", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.2", "@babel/runtime@^7.6.3", "@babel/runtime@^7.8.7":
```
Comment (hseyyidkaplan) [inline frontend/yarn.lock:178]: Is this changes special to your personal PC architecture? If it is, you can ignore this changes while pushing.

File: frontend/pages/space/[id].js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect, useCallback } from "react";
```
Comment (hseyyidkaplan) [inline frontend/pages/space/[id].js:1]: Why did you name this file as [id].js?

File: frontend/yarn.lock [hunk 3]
```diff
@@ -503,9 +503,9 @@
   dependencies:
     "glob" "7.1.7"
 
-"@next/swc-darwin-x64@13.0.4":
-  "integrity" "sha512-HK4b2rFiju8d40GTL/jH9U6OQ7BYA2MeEHs7Dm7Rp7kwQtLzP3z6osdQS8er20tIFHDE4b+oVBy03ZUQkHf0Pg=="
-  "resolved" "https://registry.npmjs.org/@next/swc-darwin-x64/-/swc-darwin-x64-13.0.4.tgz"
+"@next/swc-darwin-arm64@13.0.4":
+  "integrity" "sha512-/lajev+9GSie+rRTl5z8skW9RJwZ+TwMKLzzM24TbDk8lUjqPTyJZ/cU0NDj8J7VQAZ6EehACSh9rcJeBRtLuA=="
```
Comment (hseyyidkaplan) [inline frontend/yarn.lock:507]: Is this sha512 key special to you or in our global use? If it is special key, it is better to not pushing them.

**Review by hseyyidkaplan — CHANGES_REQUESTED** [review]
It is better to rename 'space' folders under both components and pages directory as 'detail'  because we are already serving to space with all other components and pages. It would more clear with this brand-new renaming.

File: frontend/yarn.lock (hunk 1 above)
Comment (kamilkorkut) [reply frontend/yarn.lock]: I fixed them.

File: frontend/yarn.lock (hunk 3 above)
Comment (kamilkorkut) [reply frontend/yarn.lock]: I also fixed it.

**Comment by kamilkorkut** [comment]
I have changed "space" folder name to "detail".

File: frontend/yarn.lock (hunk 1 above)
Comment (hseyyidkaplan) [reply frontend/yarn.lock]: Thanks!

File: frontend/pages/space/[id].js (hunk 2 above)
Comment (hseyyidkaplan) [reply frontend/pages/space/[id].js]: Hmm, ok, got it.

**Comment by hseyyidkaplan** [comment]
I checked all of the contributions and it seems OK. So, I merge the PR with [frontend](https://github.com/bounswe/bounswe2022group1/tree/frontend) branch.

**Pull request merged**